src/centrality.c

```diff
@@ -2445,10 +2445,8 @@ int igraph_edge_betweenness_estimate(const igraph_t *graph, igraph_vector_t *res
             neip = igraph_inclist_get(elist_out_p, actnode);
             neino = igraph_vector_int_size(neip);
             for (i = 0; i < neino; i++) {
-                igraph_integer_t edge = (igraph_integer_t) VECTOR(*neip)[i], from, to;
-                long int neighbor;
-                igraph_edge(graph, edge, &from, &to);
-                neighbor = actnode != from ? from : to;
+                igraph_integer_t edge = (igraph_integer_t) VECTOR(*neip)[i];
```

Comment:
I think the `igraph_integer_t` cast is unnecessary here, since `neip` is a `igraph_vector_int_t*`. Or am I missing something here?

Reply:
(The same thing also occurs elsewhere btw, but I first wanted to make sure this is correct).

Reply:
There are probably many other places, I only looked for `IGRAPH_OTHER()` instances and removed the explicit cast from the edge ID there (because `IGRAPH_OTHER()` is casting internally anyway).